apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html

```diff
@@ -0,0 +1,87 @@
+<div
+    tuiAppearance="whiteblock"
+    tuiCardLarge="normal"
+>
+    <tui-tabs tuiFade>
+        <button
+            tuiTab
+            (click)="updateFades(true)"
+        >
+            <header tuiHeader>
+                <h2
+                    tuiTitle
+                    [style.text-align]="'center'"
+                >
+                    Contacts
+                </h2>
+            </header>
+        </button>
+        <button
+            tuiTab
+            (click)="updateFades(false)"
+        >
+            <header tuiHeader>
+                <h2
+                    tuiTitle
+                    [style.text-align]="'center'"
+                >
+                    Recent
+                </h2>
+            </header>
+        </button>
+    </tui-tabs>
+    <tui-scrollbar>
+        @if (contacts) {
```

Comment:
let's use `if` inside cdk-virtual-scroll

Reply:
If I do this, the items will only update once

Reply:
cdkVirtualFor works a little differently

Reply:
ok, let's leave it like this for now